API/Controllers/PlatformStatsController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using API.DTOs;
+using API.Services.Interfaces;
+using Asp.Versioning;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.OutputCaching;
+
+namespace API.Controllers;
+
+[ApiController]
+[ApiVersion(1)]
+[Route("api/v{version:apiVersion}/stats")]
+public class PlatformStatsController(IPlatformStatsService statsService) : Controller
+{
+    private const int StatsCacheDurationSeconds = 60 * 10;
```

Comment:
```suggestion
    private const int StatsCacheDurationSeconds = 600;
```

These stats update very infrequently but this should be okay for now.

Reply:
> These stats update very infrequently but this should be okay for now.

yeah, just a number out of head 😁 

why the multiplication should be removed though? this way of writing time/memory constants is much better and readable imo

Reply:
Less readable to me, I dont love splitting them. Its easier to do 600/10 in my head than it is to read 60 * 10